SPECS/openssl/openssl.spec

```diff
@@ -42,17 +39,22 @@ Patch19:        openssl-1.1.1-sp80056arev3.patch
 Patch20:        openssl-1.1.1-jitterentropy.patch
 Patch21:        openssl-1.1.1-drbg-seed.patch
 Patch22:        openssl-1.1.1-fips-SymCrypt.patch
+# Don't depend on bash by default
+%define __requires_exclude ^/(bin|usr/bin).*$
+%define soversion 1.1
```

Comment:
Could you move these back to the top of the file? They are a lot easier to find there.:)

Reply:
I think this is a linter recommendation that all %defines are moved after patches. We should fix this check

Reply:
Sure moving it back up :)